source/agora/common/Task.d

```diff
@@ -51,4 +58,46 @@ public class TaskManager
         static import vibe.core.core;
         vibe.core.core.sleep(dur);
     }
+
+    /***************************************************************************
+
+        Run an asynchronous task after a given time.
+
+        The task will first run after the given `timeout`, and
+        can either repeat or run only once (the default).
+        Use the `setTimer` vibe.d's
+
+        Params:
+            timeout = Determines the minimum amount of time that elapses before
+                the timer fires.
+            dg = If non-null, this delegate will be called when the timer fires
+            periodic = Speficies if the timer fires repeatedly or only once
+
+    ***************************************************************************/
+
+    public void timer (Duration timeout, void delegate() dg,
+        Periodic periodic = Periodic.No)
+    {
+        static import vibe.core.core;
+        vibe.core.core.setTimer(timeout, dg, periodic);
+    }
+}
+
+/// Test timer
```

Comment:
Unittests should not do I/O, and vibe.d does I/O.

This test might belong to `LocalRestTaskManager`.

Reply:
https://github.com/Geod24/localrest/blob/2cf119e759cdbbe2879bf65435efd18084620498/source/geod24/LocalRest.d#L1878-L1919
The test is being performed in the LocalRest, so delete it.